modules/photons_control/clean.py

```diff
@@ -0,0 +1,188 @@
+from photons_control.script import FromGenerator
+
+from photons_app.tasks import task_register as task
+from photons_app.errors import PhotonsAppError
+
+from photons_messages import LightMessages, DeviceMessages
+
+
+def CleanConfig(indication=None, duration=None, **kwargs):
+    """
+    Returns a valid message that will either get or set the default clean cycle
```

Comment:
```suggestion
    Sends a message that will either get or set the default clean cycle
```

Reply:
Technically, this doesn't send the message. It just returns the right message back to the calling function which actually sends it.

Reply:
oh yeah, you're right. I'm still eating breakfast hahhaha

Reply:
this is what I get for pressing add comment instead of start review like I thought I had

Reply:
It'll be more obvious once I replace the `GetVersion()` method of checking for HEV support with something a little more elegant.